tests/test_datasets/test_transforms/test_loading.py

```diff
@@ -8,13 +8,15 @@
 
 import mmcv
```

Comment:
In unit tests, I suggest keeping the original UT that does not use boxlist unchanged, then adding UT of boxlists, so that we can ensure the functionality for ndarray.